app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
     def releaseTemplate = {
         shrinkResources true
         minifyEnabled true
-        proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
I would assume `proguard-android.txt` and `proguard-android-optimize.txt` don't both need to be used – is there any specific reason you chose to do that? Also, I'd prefer not to merge this file with our proguard config. What if we did:
```
proguardFiles 'proguard-android-optimize-3.5.0-modified.txt', 'proguard-rules.pro'
```

Instead?

Reply:
That makes much more sense indeed.